charmcraft/templates/init-simple/tests/spread/lib/tools/retry.j2

```diff
@@ -0,0 +1,158 @@
+#!/usr/bin/env python3
```

Comment:
Does `init-machine` need this script too?

Are these identical?
- `charmcraft/templates/init-simple/tests/spread/lib/tools/retry.j2`
- charmcraft/templates/init-kubernetes/tests/spread/lib/tools/retry.j2

If so, is it possible to symlink these files?

Reply:
It is, good idea.